.github/workflows/cla.yml

```
@@ -10,13 +10,11 @@ on:
 permissions:
   contents: write
   pull-requests: write
-
 jobs:
-  check:
+  cla-check:
```

Unfortunately, we cannot rename this job without too much unnecessary overhead. The main reason is, it is required by the PRs and if we change it, all the open PRs will have to be rebased on this change to be able to merge them.
```suggestion
  check:
```